introduction.md

```diff
@@ -4,3 +4,9 @@ Introduction
 This repository contains a bunch of files for the Deadly Vipers dojo, which is a part of Mastering Github course I took on Code School. You can safely move along, nothing interesting to see here.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?